README.md

```diff
@@ -84,24 +102,86 @@ ngrok http 4000
 ```
 
 
+## Use Cases
 
+There are two types of actors in LaraMessaging:
 
+- Account Holder: There is only one account holder. The Account holder can see and read the messages from the anonymous users without knowing who they are.
```

Comment:
It's probably important to state here that there may be many anon users talking to one account holder